### PR TITLE
Hoenn sprites update

### PR DESCRIPTION
**Pull request by DraKay**
Change the few sprites that still were too small for 80 x 80 sprites based upon Oras. So sorry for that poeple who prefer red haired water evil trainers and grants to the people who want a proper Thabita sprite to hate.

### CODE REVIEW

File: src/modules/Credits.ts [hunk 1]
```diff
@@ -215,6 +215,8 @@ export const SpriteCredits: Credit[] = [
             'Team Flare Admin (male)',
             'Team Flare Admin (female)',
             'Team Flare Aliana',
+            'Team Magma and Aqua Admins', 
+            'Team Magma Grunts'
```
Comment (Aegyo) [inline src/modules/Credits.ts:219]:
```suggestion
            'Team Magma Grunts' ,
```

**Comment by DraKay** [comment]
Upssy, sorry for the missing colon, problems of make things at almost mifnight

File: src/modules/Credits.ts [hunk 2]
```diff
@@ -215,6 +215,8 @@ export const SpriteCredits: Credit[] = [
             'Team Flare Admin (male)',
             'Team Flare Admin (female)',
             'Team Flare Aliana',
+            'Team Magma and Aqua Admins',
```
Comment (Aegyo) [inline src/modules/Credits.ts:218]:
```suggestion
            'Team Magma and Aqua Admins',
```

**Review by Jaaslet — APPROVED** [review]
Looks good. Thanks for the PR

**Pull request merged**